tests/test_optics.py

```diff
@@ -99,7 +99,16 @@ def test_OpticalPSF_flux():
     do_pickle(optics_test._psf)
     do_pickle(optics_test._psf, lambda x: x.drawImage(nx=20, ny=20, scale=1.7, method='no_pixel'))
     check_basic(optics_test, "OpticalPSF")
-    assert optics_test._screens.r0_500_effective is None
+    assert optics_test._psf._screen_list.r0_500_effective is None
```

Comment:
You removed all the places where the `screen_list` property was used in the main code.  That's the drop in coverage.  You could remedy that by switching this to use the non-underscore version.
Or if you want, you could add a real unit test that checks that `psfscreen.psf.screen_list` is what you expect for some initialization where you know what it should be.